tests/test_default_headers.py

```diff
@@ -1,9 +1,16 @@
+import asyncio
+import datetime as dt
+
+import anyio
+import anyio.abc
 import httpx
 import pytest
 
 from tests.utils import run_server
 from uvicorn import Config
 
+DATE_FORMAT = "%a, %d %b %Y %H:%M:%S GMT"
```

Comment:
what about we use the same format mechanism as in the codebase, iirc email.formatdate ?

Reply:
Going to change it